lib/octokit/client/reactions.rb

```diff
@@ -137,16 +137,18 @@ def create_pull_request_review_comment_reaction(repo, id, reaction, options = {}
 
       # Delete a reaction
       #
-      # @param id [Integer] Reaction id
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param issue_id [Integer] The Issue comment id
+      # @param reaction_id [Integer] The Reaction id
       #
       # @see https://developer.github.com/v3/reactions/#delete-a-reaction
       #
       # @example
-      #   @client.delete_reaction(1)
+      #   @client.delete_reaction("octokit/octokit.rb", 1, 2)
       #
       # @return [Boolean] Return true if reaction was deleted, false otherwise.
-      def delete_reaction(id, options = {})
-        boolean_from_response :delete, "reactions/#{id}", options
+      def delete_reaction(repo, issue_id, reaction_id, options = {})
```

Comment:
```suggestion
      def delete_issue_reaction(repo, issue_id, reaction_id, options = {})
```

Since we are changing the signature here - it's effectively a breaking change.  I'd favor being much more esorteric with the method name as will.  Much like we do for [create](https://github.com/octokit/octokit.rb/blob/33036f8695d0a9ba559c9468fb57f4cbaa1673b4/lib/octokit/client/reactions.rb#L67C11-L67C32).  Note that we effectively have 7 different "delete reaction" endpoints in the API.